### PR TITLE
Update app-center-ux-guidelines-enterprise.markdown

### PR DESCRIPTION
**Pull request by dstaab**

### CODE REVIEW

File: src/manage-apps/go-market-docs/app-center-ux-guidelines-enterprise.markdown [hunk 1]
```diff
@@ -115,7 +115,9 @@ IMPORTANT: Any administrator can connect to your application from the SAP Concur
 
 #### <a name="landing-page-sign-in-or-up"></a>Sign In or Sign Up
 
-You should use this flow if you have an online account registration form.
+This is not the most common option to use, but this flow can be used if you have an online account registration form and/or you support free trials.
```
Comment (charlieconcur) [inline src/manage-apps/go-market-docs/app-center-ux-guidelines-enterprise.markdown]: Phrase this in a positive rather than negative manner.

**Review by charlieconcur — CHANGES_REQUESTED** [review]

**Comment by charlieconcur** [comment]
We need to always phrase things in the positive direction: "Partners should do X" rather than "Partners should not do X".

@brittnwestSAP should review all of this and request changes or approve.

File: src/manage-apps/go-market-docs/app-center-ux-guidelines-enterprise.markdown [hunk 2]
```diff
@@ -177,6 +179,7 @@ This section provides a summarized checklist of the required and recommended com
   * Must clearly indicate this is an integration with Concur (required)
   * Must include links to your terms and conditions and privacy policy (required)
   * Verification apps, should provide a link to the inquiry form (strongly recommended)
+  * The Partner application should not exchange the Request token for the Access token until the User has completed all of the actions on your Landing Page.
```
Comment (brittnwest) [inline src/manage-apps/go-market-docs/app-center-ux-guidelines-enterprise.markdown:182]: Suggested text: "Your application should only exchange the request token for the access token once the user has completed all actions on your landing page (required)."

I assume this also applies to end user apps and we should add this to the "consumer" version as well?

File: src/manage-apps/go-market-docs/app-center-ux-guidelines-enterprise.markdown [hunk 3]
```diff
@@ -115,7 +115,9 @@ IMPORTANT: Any administrator can connect to your application from the SAP Concur
 
 #### <a name="landing-page-sign-in-or-up"></a>Sign In or Sign Up
 
-You should use this flow if you have an online account registration form.
+This is not the most common option to use, but this flow can be used if you have an online account registration form and/or you support free trials.
+
+> Note: If you support Free Trials, your Landing Page is required to indicate how long the Trial will last and what the Partner will do with the data collected if the customer declines to move forward as a customer with the Partner. This text is in addition to the other requirements such as checkboxes for the user to select for your Privacy Policy and Terms and Conditions.
```
Comment (brittnwest) [inline src/manage-apps/go-market-docs/app-center-ux-guidelines-enterprise.markdown:120]: Suggested text: "Note: if you support free trials, your landing page must also indicate the trial duration and how the data collected will be handled should the customer cancel following the free trial period."

The privacy policy and T&Cs should be handled in a separate bullet if we want to call that out.

File: src/manage-apps/go-market-docs/app-center-ux-guidelines-enterprise.markdown (hunk 1 above)
Comment (brittnwest) [reply src/manage-apps/go-market-docs/app-center-ux-guidelines-enterprise.markdown]: Suggested text: "Use this option if you support online account registration or free trials."

File: src/manage-apps/go-market-docs/app-center-ux-guidelines-enterprise.markdown [hunk 4]
```diff
@@ -104,7 +104,7 @@ Sign In or Sign Up| This is used when the customer may not have an account or re
...
+This is the recommended option for the majority of Enterprise applications since the partner will likely have had contract negotiations with the customer outside of the Concur connection flow.  Once the contract is finalized with the customer, they will have the value to copy and paste into the Verfication field of your Landing Page. If your application activation fits this description and requires pre-requisites (e.g. a contract or other manual set up is required), you may introduce an additional verification step. Once all pre-requisites are complete and prior to the user activating the application, you will supply the administrator with a verification code. Once the user connects from the SAP Concur App Center, your landing page will prompt the user for a verification code; the user must provide a verification code to proceed. In the remainder of this guide, this will be referred to as a “Verification” app. This page should clearly articulate the next steps and should provide the user the option to:
```
Comment (brittnwest) [inline src/manage-apps/go-market-docs/app-center-ux-guidelines-enterprise.markdown]: Suggested text: "This is the recommended option for paid Enterprise applications. Once the contract has been signed by the customer, you will provide the customer with a verification code to enter on your landing page. In the remainder of this guide..."

**Review by brittnwest — CHANGES_REQUESTED** [review]

File: src/manage-apps/go-market-docs/app-center-ux-guidelines-enterprise.markdown (hunk 1 above)
Comment (dstaab) [reply src/manage-apps/go-market-docs/app-center-ux-guidelines-enterprise.markdown]: sounds good.

File: src/manage-apps/go-market-docs/app-center-ux-guidelines-enterprise.markdown [hunk 5]
```diff
@@ -104,7 +104,7 @@ Sign In or Sign Up| This is used when the customer may not have an account or re
 
 #### <a name="landing-page-verification"></a>Verification
 
-If your application activation requires pre-requisites (e.g. a contract or other manual set up is required), you may introduce an additional verification step. Once all pre-requisites are complete and prior to the user activating the application, you will supply the administrator with a verification code. Once the user connects from the SAP Concur App Center, your landing page will prompt the user for a verification code; the user must provide a verification code to proceed. In the remainder of this guide, this will be referred to as a “Verification” app. This page should clearly articulate the next steps and should provide the user the option to:
+This is the recommended option for Enterprise applications that already have a signed Agreement in place between the Partner and the customer. Once the contract has been signed by the customer, you will provide the customer with a verification code to enter on your landing page. In the remainder of this guide, this will be referred to as a “Verification” app. This page should clearly articulate the next steps and should provide the user the option to:
```
Comment (retrosight) [inline src/manage-apps/go-market-docs/app-center-ux-guidelines-enterprise.markdown:107]: Test review from a Write account.

**Review by retrosight — APPROVED** [review]

File: src/manage-apps/go-market-docs/app-center-ux-guidelines-enterprise.markdown [hunk 6]
```diff
@@ -115,7 +115,9 @@ IMPORTANT: Any administrator can connect to your application from the SAP Concur
 
 #### <a name="landing-page-sign-in-or-up"></a>Sign In or Sign Up
 
-You should use this flow if you have an online account registration form.
+Use this option if you support online account registration or free trials.
+
+> Note: If you support Free Trials, your Landing Page is required to indicate how long the Trial will last and what the Partner will do with the data collected if the customer declines to move forward as a customer with the Partner. This text is in addition to the other requirements such as checkboxes for the user to select for your Privacy Policy and Terms and Conditions.
```
Comment (retrosight) [inline src/manage-apps/go-market-docs/app-center-ux-guidelines-enterprise.markdown:120]: Another test review comment.

**Review by retrosight — CHANGES_REQUESTED** [review]

**Review by charlieconcur — APPROVED** [review]

**Comment by charlieconcur** [comment]
@brittnwestSAP Waiting for you to give the thumbs up.

**Comment by charlieconcur** [comment]
Slack: Brittany West [8:42 AM] Yes, looks good now.

**Pull request merged**